src/callbacks_step/analysis_dg1d.jl

```diff
@@ -176,6 +227,16 @@ function integrate(func::Func, u,
 end
 
 
+function integrate_element(func::Func, u, element,
+                           mesh::Union{TreeMesh{1},StructuredMesh{1}},
+                           equations, dg::DGSEM, cache; normalize=true) where {Func}
+  integrate_element_via_indices(u, element, mesh, equations, dg, cache; normalize=normalize) do u, i, dummy, equations, dg
```

Comment:
`dummy` replaces `element` in https://github.com/trixi-framework/Trixi.jl/blob/b7484001bcefa6ffa119e2ff5b5f7228f2e201a8/src/callbacks_step/analysis_dg1d.jl#L182

- this is probably not sophisticated code, but it was the only way I managed to get this working.